Web/starter_project_iconics/components/profile/AccountInfo.tsx

```diff
@@ -0,0 +1,124 @@
+import { AiFillEye } from 'react-icons/ai'
+import React, { useState } from 'react'
+
+const AccountInfo: React.FC = () => {
+  const [old, setOld] = useState<string>('')
+  const [oldShow, setOldShow] = useState<boolean>(false)
+  const [newPassword, setNewPassword] = useState<string>('')
+  const [newPasswordShow, setNewPasswordShow] = useState<boolean>(false)
+  const [confirmed, setConfirmed] = useState<string>('')
+  const [confirmedShow, setConfirmedShow] = useState<boolean>(false)
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+    // Do something with form data, e.g. submit to a server
```

Comment:
It would be good If you add this comment at the beginning of the function instead of inside it.